webapp/components/product-info.js

```diff
@@ -159,6 +163,30 @@ const ProductInfo = (superClass) => class extends superClass {
   }
 };
 
+// https://stackoverflow.com/questions/5916900/how-can-you-detect-the-version-of-a-browser
+function getActiveBrowser() {
+  const ua = navigator.userAgent;
+  let tem, M = ua.match(/(opera|chrome|safari|firefox|msie|trident(?=\/))\/?\s*(\d+)/i) || [];
+  if(/trident/i.test(M[1])){
+    tem = /\brv[ :]+(\d+)/g.exec(ua) || [];
+    return { name:'IE', version: tem[1] || '' };
+  }
+  if (M[1] === 'Chrome') {
+    tem = ua.match(/\bOPR|Edge\/(\d+)/);
+    if (tem !== null) {
+      return { name: 'Opera', version: tem[1] };
```

Comment:
Looks like this will say that Edge is Opera?

Rather than trying to parse out the browser and misclassify some, can the help text just say "your browser"?

Reply:
Done.